addons/calendar/models/calendar_recurrence.py

```diff
@@ -104,13 +106,13 @@ class RecurrenceRule(models.Model):
     end_type = fields.Selection(END_TYPE_SELECTION, default='count')
     interval = fields.Integer(default=1)
     count = fields.Integer(default=1)
-    mo = fields.Boolean()
-    tu = fields.Boolean()
-    we = fields.Boolean()
-    th = fields.Boolean()
-    fr = fields.Boolean()
-    sa = fields.Boolean()
-    su = fields.Boolean()
+    mon = fields.Boolean()
+    tue = fields.Boolean()
+    wed = fields.Boolean()
+    thu = fields.Boolean()
+    fri = fields.Boolean()
+    sat = fields.Boolean()
+    sun = fields.Boolean()
```

Comment:
The string could be harmonized, see lunch_alert.py

Reply:
@arj-odoo If I do not give string attribute then framework will automatically give string attribute with capital latter.

Reply:
> @arj-odoo If I do not give string attribute then framework will automatically give string attribute with capital latter.

Sure, then you don't need the string anymore in Lunch :-).